docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers mLRS in its core use case to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use OpenTX or EdgeTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
+
+The steps here should be followed in order and when practical we provide a way to confirm a step was completed successfully.
+
+We expect that you are starting fresh and that all ArduPilot and mLRS parameters not covered here still have their default settings.
```

Comment:
"We further assume that ..."

I would move this sentence one up, to where the other assumptions are